.github/workflows/lock.yml

```diff
@@ -0,0 +1,60 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '15 4 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        id: lock-threads
+        with:
+          # Number of days of inactivity before a closed issue is locked
+          issue-lock-inactive-days: '365'
+          # Do not lock issues created before a given timestamp, value must follow ISO 8601
+          issue-exclude-created-before: ''
+          # Do not lock issues with these labels, value must be a comma separated list of labels or ''
+          issue-exclude-labels: ''
+          # Labels to add before locking an issue, value must be a comma separated list of labels or ''
+          issue-lock-labels: ''
+          # Comment to post before locking an issue
+          issue-lock-comment: >
+            Hi there, thank you for your contribution to Conda!
+
+            This issue has been automatically locked since it
+            has not had recent activity after it was closed.
+
+            Please open a new issue for related bugs.
+          # Reason for locking an issue, value must be one of resolved, off-topic, too heated, spam or ''
+          issue-lock-reason: 'resolved'
+
+          # Number of days of inactivity before a closed pull request is locked
+          pr-lock-inactive-days: '365'
+          # Do not lock pull requests created before a given timestamp, value must follow ISO 8601
+          pr-exclude-created-before: ''
+          # Do not lock pull requests with these labels, value must be a comma separated list of labels or ''
+          pr-exclude-labels: ''
+          # Labels to add before locking a pull request, value must be a comma separated list of labels or ''
+          pr-lock-labels: ''
```

Comment:
We should probably have some sort of lock label, just so that we can easily find some data on this in the future.

Reply:
Good idea, I've created a new "locked" label and updated the config.